cmd/benchmark.go

```diff
@@ -15,6 +15,14 @@ import (
 func main() {
 	runs := flag.Int("runs", 100, "number of runs per benchmark")
 	profile := flag.String("profile", "benchmark", "profile to use for minikube commands")
+	// images is a list of images we use, seperated by ,
+	images := flag.String("images", "", "a comma separated list of images to benchmark")
+
+	benchFlows := flag.String("iters", "iterative,non-iterative", "a comma separated list of flows to benchmark, options [iterative,non-iterative]")
+
+	benchMethods := flag.String("bench-methods", "", "a comma separated list of benchmark method names")
+	memory := flag.String("memory", "", "Amount of RAM to allocate to Kubernetes (format: <number>[<unit>], where unit = b, k, m or g). Use \"max\" to use the maximum amount of memory")
```

Comment:
```suggestion
	images := flag.String("images", "", "a comma separated list of images to benchmark")
	benchFlows := flag.String("iters", "iterative,non-iterative", "a comma separated list of flows to benchmark, options [iterative,non-iterative]")
	benchMethods := flag.String("bench-methods", "", "a comma separated list of benchmark method names")
	memory := flag.String("memory", "", "Amount of RAM to allocate to Kubernetes (format: <number>[<unit>], where unit = b, k, m or g). Use \"max\" to use the maximum amount of memory")
```
Consistent spacing between flags